README.org

```diff
@@ -183,182 +296,32 @@ for records, if you need them.
-          {:sync? true})
-#+END_SRC
-
-
-In a ClojureScript REPL you can evaluate the expressions from the REPL
-each wrapped in a go-block.
-
-For simple purposes a memory store wrapping an Atom is implemented as
-well: 
-
-#+BEGIN_SRC clojure 
-  (ns test-db
-    (:require [konserve.memory :refer [new-mem-store]]
-              [konserve.core :as k]))
-
-  (go (def my-db (<! (new-mem-store)))) ;; or (go (def my-db (<!
-  (new-mem-store (atom {:foo 42}))))) 
-#+END_SRC
-
-In ClojureScript from a browser (you need IndexedDB available in your js
-env): 
-
-#+BEGIN_SRC clojure
-  (ns test-db
-    (:require [konserve.indexeddb :refer [new-indexeddb-store]])
-    (:require-macros [cljs.core.async.macros :refer [go go-loop]]))
-
-  (go (def my-db (<! (new-indexeddb-store "konserve"))))
-
-  (go (println "get:" (<! (k/get-in my-db ["test" :a]))))
-
-  (go (doseq [i (range 10)] (<! (k/assoc-in my-db [i] i))))
-
-  ;; prints 0 to 9 each on a line
-  (go (doseq [i (range 10)] (println (<! (k/get-in my-db [i])))))
-
-  (go (println (<! (k/assoc-in my-db ["test"] {:a 1 :b 4.2}))))
-
-  (go (println (<! (k/update-in my-db ["test" :a] inc))))
-  ;; => "test" contains {:a 2 :b 4.2}
-#+END_SRC
-
-For non-REPL code execution you have to put all channel operations in one
-top-level go-block for them to be synchronized:
-
-#+BEGIN_SRC clojure 
-  (ns test-db
-    (:require [konserve.indexeddb :refer [new-indexeddb-store]])
-    (:require-macros [cljs.core.async.macros :refer [go go-loop]]))
-
-  (go (def my-db (<! (new-indexeddb-store "konserve")))
-
-      (println "get:" (<! (k/get-in my-db ["test" :a])))
-
-      (doseq [i (range 10)]
-         (<! (k/assoc-in my-db [i] i))))
-
-#+END_SRC
-
-For more examples have a look at the comment blocks at the end of the
-respective namespaces.
-
 ** Backend implementation guide
    :PROPERTIES:
    :CUSTOM_ID: h:7582b1c9-e305-4d51-a808-c10eb447f3de
    :END:
-   
-   We provide a [[file:doc/backend.org][backend implementation guide]] .
-   
-** JavaScript bindings
-   :PROPERTIES:
-   :CUSTOM_ID: h:05f3f415-afc2-4e11-a9f6-6e25519adb12
-   :END:
 
-There are experimental javascript bindings in the =konserve.js=
-namespace:
-
-#+BEGIN_SRC javascript
-    goog.require("konserve.js");
-
-    konserve.js.new_mem_store(function(s) { store = s; });
-    # or
-    konserve.js.new_indexeddb_store("test_store", function(s) { store = s; })
-
-    konserve.js.exists(store, ["foo"], function(v) { console.log(v); });
-    konserve.js.assoc_in(store, ["foo"], 42, function(v) {});
-    konserve.js.get_in(store,
-                       ["foo"],
-                       function(v) { console.log(v); });
-    konserve.js.update_in(store,
-                          ["foo"],
-                          function(v) { return v+1; },
-                          function(res) { console.log("Result:", res); });
-#+END_SRC
+   We provide a [[file:doc/backend.org][backend implementation guide]] .
 
 ** TODO
```

Comment:
For some reason this headline doesn't get shown when I look at the rendered version on GitHub. Any ideas why?

Reply:
Because capitalized TODO is a keyword in org-mode to annotate headlines as a todo item. I changed it.